content/concepts/peers.md

```diff
@@ -107,3 +106,35 @@ their addresses using [peer routing](/concepts/peer-routing/).
 [definition_multihash]: /reference/glossary/#multihash
 
 [spec_peerid]: https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md
+
+## PeerStore
```

Comment:
This seems overly specific to the Go implementation. Would it be sufficient to just say that a node will typically have a temporary store to store peer keys, addresses and associated metadata.

Reply:
Updated